plugins/modules/ini_file.py

```diff
@@ -46,6 +46,24 @@
       - If left empty, being omitted, or being set to V(null), the O(option) will be placed before the first O(section).
       - Using V(null) is also required if the config format does not support sections.
     type: str
+  section_has:
+    type: list
+    elements: dict
+    required: false
+    suboptions:
+      option:
+        type: str
+        description: The option to look for within the section.
+        required: true
+      value:
+        type: str
+        description: Locate the section with this specific value.
+        required: true
+    description:
+      - Among possibly multiple sections of the same name, select the one that contains these values.
+      - With O(state=present), if a suitable section is not found, a new section will be added, including the required options.
+      - With O(state=absent), at most one O(section) is removed if it contains the values.
+    version_added: 8.0.0
```

Comment:
8.0.0 has already been released; the next release accepting new features will be 8.1.0:
```suggestion
    version_added: 8.1.0
```